prow/gerrit/client/client.go

```diff
@@ -122,6 +122,20 @@ type RevisionInfo = gerrit.RevisionInfo
 // FileInfo is a gerrit.FileInfo
 type FileInfo = gerrit.FileInfo
 
+// Map from instance name to repos to lastsync time for that repo
+type LastSyncState map[string]map[string]time.Time
+
+func (l *LastSyncState) DeepCopy() LastSyncState {
+	result := LastSyncState{}
```

Comment:
`var result LastSyncState` please (and elsewhere)

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

Reply:
won't that be nil? I'm allocating new values to it immediately after.

Reply:
Oh right its a map